src/tools/crossgen2/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1842,5 +1842,7 @@ private bool TypeRequiresMarshaling(TypeDesc type, bool isReturnType)
 
             return false;
         }
+
+        private int SizeOfPInvokeTransitionFrame => TransitionBlock.FromTarget(_compilation.NodeFactory.Target).SizeOfPInvokeTransitionBlock;
```

Comment:
This should be hardcoded to `TARGET_POINTER_SIZE * 11`, like what it is in crossgen in R2R. It is a bit more than what's required. We do want to be hardcoding the exact layout in R2R.